File: java-junit/src/test/java/org/codecop/Session1_GreeterTest.java
package org.codecop;

import static org.junit.Assert.assertEquals;

import org.junit.Test;

/**
 * Session 1: GreeterTest - Your first tests.
 */
public class Session1_GreeterTest {

    // TODO We will add the proper assertions together.

    @Test
    public void shouldReturnHelloName() {
        Greeter greeter = new Greeter();
        // TODO Check that "Hello Peter", greeter.greet("Peter").
    }

    @Test
    public void shouldReturnHelloForNull() {
        Greeter greeter = new Greeter();
        // TODO Check that "Hello", greeter.greet(null).
    }

    @Test
    public void shouldIgnoreWhitespace() {
        Greeter greeter = new Greeter();
        // TODO Check that "Hello Peter", greeter.greet(" Peter ").
    }

}

File: csharp-xunit/WordCount.Tests/Session1_GreeterTest.cs
using Xunit;

namespace Org.Codecop.WordCount.Tests
{
    /// <summary>Session 1: GreeterTest - Your first tests.</summary>
    public class Session1_GreeterTest
    {

        // TODO We will add the proper assertions together.

        [Fact]
        public void ShouldReturnHelloName()
        {
            var greeter = new Greeter();
            // TODO Check that "Hello Peter" and greeter.Greet("Peter") are equal.
        }

        [Fact]
        public void ShouldReturnHelloForNull()
        {
            var greeter = new Greeter();
            // TODO Check that "Hello" and greeter.Greet(null) are equal.
        }

        [Fact]
        public void ShouldIgnoreWhitespace()
        {
            var greeter = new Greeter();
            // TODO Check that "Hello Peter" and greeter.Greet(" Peter ") are equal.
        }
    }
}
